### PR TITLE
VAA 5.1 comleted

### PR DESCRIPTION
**Pull request by RaLLy08**

### CODE REVIEW

File: src/Controller.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+        this.init();
+    }
+
+    init = () => {
+        this._view.init();
+    }
+}
+
+export default Controller;
```
Comment (hnamazli) [inline src/Controller.js:15]: The task seems to be under the database why is the controller here?

File: src/index.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
+import Controller from './Controller';
 
 const init = () => {
     const view = new View();
-
-    view.init();
-};
+    const model = new Model();
+    const controller =  new Controller(view, model);
+    window.model = model;
```
Comment (hnamazli) [inline src/index.js]: not flooded with test maids

File: src/view.js [hunk 3]
```diff
@@ -3,7 +3,8 @@ class View {
         this._root = document.getElementById('root');
 
     }
-
+    
+
```
Comment (hnamazli) [inline src/view.js]: code style

File: src/index.js [hunk 4]
```diff
@@ -1,11 +1,14 @@
 import './style.scss';
-import View from './View';
+import View from './view';
+import Model from './Model';
+import Controller from './Controller';
```
Comment (hnamazli) [inline src/index.js:4]: not for task

File: src/index.js [hunk 5]
```diff
@@ -1,11 +1,14 @@
...
 
 const init = () => {
     const view = new View();
-
-    view.init();
-};
+    const model = new Model();
+    const controller =  new Controller(view, model);
```
Comment (hnamazli) [inline src/index.js:9]: not for task

**Review by hnamazli — COMMENTED** [review]
I think it’s not done to task, and it’s important that the test data should not get into production

**Review by Musa-94 — APPROVED** [review]

**Review by hnamazli — APPROVED** [review]

**Pull request merged**